cli/peerid.go

```diff
@@ -1,101 +0,0 @@
-// Copyright 2022 Democratized Data Foundation
-//
-// Use of this software is governed by the Business Source License
-// included in the file licenses/BSL.txt.
-//
-// As of the Change Date specified in that file, in accordance with
-// the Business Source License, use of this software will be governed
-// by the Apache License, Version 2.0, included in the file
-// licenses/APL.txt.
-
-package cli
-
-import (
-	"encoding/json"
-	"io"
-	"net/http"
-	"os"
-
-	"github.com/spf13/cobra"
-
-	httpapi "github.com/sourcenetwork/defradb/api/http"
-	"github.com/sourcenetwork/defradb/config"
-	"github.com/sourcenetwork/defradb/errors"
-)
-
-func MakePeerIDCommand(cfg *config.Config) *cobra.Command {
```

Comment:
question: Is this now provided by p2p_info?

Reply:
Yes, as a follow up I will be adding the multiaddresses as a return value there as well.